test/test_client.py

```diff
@@ -0,0 +1,246 @@
+    with pytest.raises(KinErrors.AccountNotFoundError):
+        test_client.get_account_data(address)
+
+    acc_data = test_client.get_account_data(test_account.get_public_address())
+    assert acc_data
+    assert acc_data.id == test_account.get_public_address()
+    assert acc_data.sequence
+    assert acc_data.data == {}
+
+    assert acc_data.thresholds
+    assert acc_data.thresholds.low_threshold == 0
+    assert acc_data.thresholds.medium_threshold == 0
+    assert acc_data.thresholds.high_threshold == 0
+
+    assert acc_data.flags
+    assert not acc_data.flags.auth_revocable
+    assert not acc_data.flags.auth_required
+
+    assert len(acc_data.balances) == 2
+    asset_balance = acc_data.balances[0]
+    native_balance = acc_data.balances[1]
+    assert asset_balance.balance > 0
+    # default limit
+    assert asset_balance.limit == 922337203685.4775807
+    assert asset_balance.asset_type == 'credit_alphanum4'
+    assert asset_balance.asset_code == 'KIN'
+    assert asset_balance.asset_issuer == test_client.kin_asset.issuer
+    assert native_balance.balance > 0
+    assert native_balance.asset_type == 'native'
+
+    # just to increase test coverage
+    assert str(acc_data)
+
+
+def test_get_transaction_data(test_client):
+    from kin import OperationTypes
+    from kin.blockchain.horizon_models import TransactionData
+
+    with pytest.raises(ValueError, match='invalid transaction hash: bad'):
+        test_client.get_transaction_data('bad')
+
+    with pytest.raises(KinErrors.ResourceNotFoundError):
+        test_client.get_transaction_data('deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef')
+
+    address = 'GAHTWFVYV4RF2AMEZP3X2VOK4HB3YOSARU7VNVTP7J2OLDSVOP564YEN'
+    tx_hash = test_client.friendbot(address)
+    sleep(5)
+    tx_data = test_client.get_transaction_data(tx_hash)
+    assert tx_data
+    assert tx_data.id == tx_hash
+    assert tx_data.timestamp
+    assert tx_data.memo is None
+    assert tx_data.operation
+    assert tx_data.source == test_client.kin_asset.issuer  # root account
+    assert tx_data.operation.type == OperationTypes.CREATE_ACCOUNT
+    assert tx_data.operation.destination == address
+    assert tx_data.operation.starting_balance == 10
+
+    tx_data = test_client.get_transaction_data(tx_hash,simple=False)
+    assert isinstance(tx_data, TransactionData)
+
+
+def test_friendbot(test_client):
+    from kin import AccountStatus
+    address = 'GDIPKVWPVCL5E5MX4UWMLCGXMDWEMEYAZGCI3TPJPVDG5ZFA6VJAA7RA'
+    test_client.friendbot(address)
+    assert test_client.get_account_status(address) == AccountStatus.NOT_ACTIVATED
+
+    with pytest.raises(ValueError):
+        test_client.friendbot('bad')
+
+
+def test_verify_kin_payment(test_client, test_account):
+    address = 'GCZXR4ILXETTNQMUNF54ILRMPEG3UTUUMYKPUXU5633VCOABZZ63H7FJ'
+    seed = 'SCJQYWGJTL3NUEJ4H2QRDGQHO4HW3RX4IAZEEUDGLTIE5I4MEFR4UZYZ'
+    tx_hash = test_client.friendbot(address)
+    sleep(5)
```

Comment:
can we create also unit tests instead of integration?
meaning mocking network result instead of actually using a network?